enzy_htp/structure/structure_io/_interface.py

```diff
@@ -0,0 +1,47 @@
+"""the module for interface definiation of strutcure file I/O classes
```

Comment:
typo of definition and structure

Reply:
Really hope I can use grammerly in vscode haha